src/libecalc/dto/components.py

```diff
@@ -137,7 +138,8 @@ class EmitterModel(EcalcBaseModel):
     emission_rate: Expression
 
     regularity: Dict[datetime, Expression]
-
+    emission_rate_type: RateType = RateType.STREAM_DAY
+    convert_to_stream_day: bool = False
```

Comment:
why is this needed?

Reply:
we should always convert to stream day...or?

Reply:
Not if the user has given stream day as input.

Reply:
no, but the "stream day timeseries" also has a `to_stream_day()` and basically returns the same value without conversion...ie the correct timeseries should be created based on the rate_type ...

Reply:
I was looking for that, but could not find it.